src/plone/restapi/tests/test_copymove.py

```diff
@@ -101,8 +62,114 @@ def test_get_object_by_uid(self):
         obj = service.get_object(self.doc1.UID())
         self.assertEqual(self.doc1, obj)
 
+
+class TestCopyMoveFunctional(unittest.TestCase):
+
+    layer = PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.app = self.layer['app']
+        self.portal = self.layer['portal']
+        self.portal_url = self.portal.absolute_url()
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
+        self.doc1 = self.portal[self.portal.invokeFactory(
+            'Document', id='doc1', title='My Document')]
+        self.doc2 = self.portal[self.portal.invokeFactory(
+            'Document', id='doc2', title='My Document')]
+        self.folder1 = self.portal[self.portal.invokeFactory(
+            'Folder', id='folder1', title='My Folder')]
+
+        api.user.create(
+            email='memberuser@example.com',
+            username='memberuser',
+            password='secret'
+        )
+
+        self.api_session = RelativeSession(self.portal_url)
+        self.api_session.headers.update({'Accept': 'application/json'})
+        self.api_session.auth = (SITE_OWNER_NAME, SITE_OWNER_PASSWORD)
+
+        transaction.commit()
+
+    def test_copy_single_object(self):
+        response = self.api_session.post(
+            '/@copy',
+            json={
+                "source": self.doc1.absolute_url()
+            }
+        )
+        transaction.commit()
```

Comment:
@sneridagh sorry for being picky. Just one very minor comment. I usually add a blank line before the assert statements to separate the preconditions/actions from the actual test. I leave it up to you if you want to adopt this...